index.html

```diff
@@ -5289,7 +5289,7 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 			<rdef>menu</rdef>
 			<div class="role-description">
 				<p>A type of <a>widget</a> that offers a list of choices to the user.</p>
-				<p>A menu is often a list of common actions or functions that the user can invoke. The <code>menu</code> <a>role</a> is appropriate when a list of menu items is presented in a manner similar to a menu on a desktop application.</p>
+				<p>A menu is a popup list of items that represent a set of actions or functions a user can invoke. The menu and list of <code>menuitems</code> are almost always directly related to the item that the user interacted with to reveal the menu. The <code>menu</code> <a>role</a> is appropriate when a list of menu items is presented in a manner similar to a menu on a desktop application. For instance, a menu could be used to represent a context menu for its invoking element, or it would be used to render sub-menus for items of a <rref>menubar</rref> or another <code>menu</code> popup.</p>
```

Comment:
I think we can further simplify and clarify this language. I believe it is also important to specify that activating menuitems close menus,  to enumerate how menus are triggered, and provide a link to APG.

```suggestion
				<p>A menu is a popup containing a set of <code>menuitem</code> elements that can be invoked to perform an action or function. The actions are nearly always directly related to the element that the user activated to popup the menu. The <code>menu</code> <a>role</a> is appropriate when a popup is presented and behaves in a manner similar to a menu on a desktop application. activating a <code>menuitem</code> both performs the action and causes the popup to close.</p>
				<p>Menu popups can be triggered by:</p>
				<ul>
					<li>Activating a <code>menuitem</code> in a parent <code>menu</code> or <rref>menubar</rref>.
					<li>Activating a <rref>button</rref> that has the <pref>aria-haspopup</pref> set to <code>true</code>.
					<li>Executing a command that triggers a context menu for an element.</li>
				</ul>
				<p>See the <a href="https://www.w3.org/WAI/ARIA/apg/patterns/menubar/">WAI-ARIA Authoring Practices</a> for more details about implementing menus.</p>
```

Reply:
Is it odd to link to the APG page that only has menubar examples, from the menu definition?

not part of my previous PR, but this suggestion made me realize do we need to change the instance of `<code>menuitem</code>` to just say "menu item" - so as to not imply that only `menuitem` elements are allowed, and not also `menuitemradio` and `menuitemcheckbox`?

For the activation of menuitems, does that belong in the `menuitem` section?  In other words, taking `menuitemcheckbox` into consideration, is it _always_ true that activating one of those should cause a menu popup to close?

Pending those above questions, I'm otherwise good with these suggestions.  I just don't know if they all belong in just this `menu` update?

Reply:
@mcking65 re-ping on my questions concerning your suggested updates, so we can move this along

Reply:
so as to move this forward again, i used some of the provided feedback, but not all.